app/views/results/common/_annotations.js.erb

```diff
@@ -229,23 +229,32 @@
     document.getElementById('new_annotation_content').focus();
   }
 
-  function get_selection_box() {
+  function get_selection_box_coordinates() {
     var box = document.getElementById('sel_box');
     var x1 = parseInt(box.style.left, 10) -
-               document.getElementById('image_preview').offsetLeft +
-               document.getElementById('image_container').scrollLeft;
+    document.getElementById('image_preview').offsetLeft +
+    document.getElementById('image_container').scrollLeft;
     var y1 = parseInt(box.style.top, 10) -
-               document.getElementById('image_preview').offsetTop +
-               document.getElementById('codeviewer').scrollTop;
+    document.getElementById('image_preview').offsetTop +
+    document.getElementById('codeviewer').scrollTop;
     var x2 = x1 + parseInt(box.style.width, 10);
     var y2 = y1 + parseInt(box.style.height, 10);
 
     if ((x2 - x1) < 1 || isNaN(x2 - x1)) {
-      alert("<%= I18n.t('marker.annotation.select_an_area') %>");
       return false;
+    } else {
+      return { x1: x1, x2: x2, y1: y1, y2: y2 };
     }
+  }
 
-    return { x1: x1, x2: x2, y1: y1, y2: y2 };
+  function get_selection_box() {
+    var coordinates = get_selection_box_coordinates();
+    if (!coordinates) {
+      alert("<%= I18n.t('marker.annotation.select_an_area') %>");
```

Comment:
This alert code was from the original source. I wanted to leave things as is to stick with the file convention, especially since` I18n` is prefixed in other areas in this file. I can remove this and other instances where this occurs in the file if that is desirable.